ironfish-cli/src/commands/wallet/multisig/dkg/create.ts

```diff
@@ -112,9 +112,13 @@ export class DkgCreateCommand extends IronfishCommand {
       })
       multisigClient.start()
 
-      if (sessionId) {
-        multisigClient.joinSession(sessionId)
-      }
+      multisigClient.onConnectedMessage.on(() => {
+        if (sessionId) {
+          Assert.isNotNull(multisigClient)
+          multisigClient.joinSession(sessionId)
+          multisigClient.onConnectedMessage.clear()
+        }
+      })
```

Comment:
Should we block on waiting for a confirmed connection? If the `connected` message gets dropped for some reason then we'll continue without ever joining the session. Maybe that's being too paranoid about dropped messages 😅